scripts/get_stress_test_summary.py

```diff
@@ -0,0 +1,62 @@
+import os
+import statistics
+from glob import glob
+from scripts.get_stats import get_stats
+
+
+def get_results(folders):
+
+    results = {
+        "get": [],
+        "post": [],
+        "average_get": [],
+        "average_post": [],
+        "average_total": [],
+        "error_percentage": 0,
+        "total_requests": 0,
+        "total_failures": 0
+    }
+
+    for folder in folders:
+        stats = get_stats(folder)
+        results["get"].extend(stats["get"])
+        results["post"].extend(stats["post"])
+        results["total_requests"] = results["total_requests"] + stats["total_requests"]
+        results["total_failures"] = results["total_failures"] + stats["total_failures"]
+
+    results["average_get"] = statistics.mean(results["get"])
+    results["average_post"] = statistics.mean(results["post"])
+    results["average_total"] = statistics.mean(results["get"] + results["post"])
+    results["error_percentage"] = (results["total_failures"] * 100) / results["total_requests"]
+
+    return results
+
+
+def parse_environment_variables():
+    output_dir = os.getenv("OUTPUT_DIR")
+
+    if not output_dir:
+        print(
+            "'OUTPUT_DIR' environment variable must be provided e.g. outputs/daily-test"
+        )
+        sys.exit(1)
```

Comment:
```suggestion
    try:
    	output_dir = os.environ["OUTPUT_DIR"]
	except KeyError:
        print(
            "'OUTPUT_DIR' environment variable must be set"
        )
        sys.exit(1)
```

Reply:
This is now inconsistent with the other place in the code base plus if we did try except, don't see a reason to sys.exit(). Also, I don't see the need for a method when we have 1 env var we need.

Reply:
@MebinAbraham Should I remove the method then?

Reply:
I would given, it was only added to the other script since there were multiple env vars.